apps/swirl-docs/src/components/HeaderNavigation/index.tsx

```diff
@@ -26,44 +27,45 @@ const HeaderNavigation = () => {
       </Link>
       <nav
         aria-label="main"
-        className="flex justify-between md:justify-start items-center h-[72px] w-full px-6 border-b-1 font-normal text-base"
+        className="flex justify-between items-center h-[72px] w-full px-6 border-b-1 font-normal text-base"
       >
-        <Link href="/">
-          <a className="flex justify-center items-center mr-8">
-            <Image
-              alt="Swirl home"
-              src="/swirl-icon-temp.svg"
-              width={32}
-              height={32}
-            />
-            <span className="font-bold ml-3">Swirl</span>
-          </a>
-        </Link>
+        <div className="inline-flex w-full justify-between md:justify-start">
+          <Link href="/">
+            <a className="flex justify-center items-center mr-8">
```

Comment:
**question:** Is the inner `<a>` still needed with Next.js 13?

Reply:
Possibly not, but this project is still in v12 😅 i wanted to do the update after the initial release, or at least when the site is ready to launch. so i think after the improvement tickets and the initial search there is room for that :)

Reply:
Sorry, assumed you were on 13.